partcad/src/partcad/project_config.py

```
@@ -34,32 +38,25 @@
         if not os.path.isfile(self.config_path):
-            pc_logging.error(
-                "PartCAD configuration file is not found: '%s'"
-                % self.config_path
-            )
+            pc_logging.error("PartCAD configuration file is not found: '%s'" % self.config_path)
             self.broken = True
             return
 
         # Read the body of the configuration file
-        fp = open(self.config_path, "r")
+        fp = open(self.config_path, "r", encoding="utf-8")
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)